madrepo.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+set -e
+echo 'deb Https://sourceforge.net/projects/madlinux/files/repo core main'|sudo tee /etc/apt/sources.list.d/madlinux.list
```

Comment:
I tried here on pop_os and got an error because of the capital H, so should be `https:` instead of `Https:`, no?

Reply:
Sorry for this, i'll fix it.